authority/provisioner/scep.go

```diff
@@ -29,9 +36,19 @@ type SCEP struct {
 	// intermediate in the GetCACerts response
 	IncludeRoot bool `json:"includeRoot,omitempty"`
 
+	// ExcludeIntermediate makes the provisioner skip the intermediate CA in the
+	// GetCACerts response
+	ExcludeIntermediate bool `json:"excludeIntermediate,omitempty"`
+
 	// MinimumPublicKeyLength is the minimum length for public keys in CSRs
 	MinimumPublicKeyLength int `json:"minimumPublicKeyLength,omitempty"`
 
+	// TODO(hs): also support a separate signer configuration?
+	DecrypterCertificate []byte `json:"decrypterCertificate,omitempty"`
+	DecrypterKeyPEM      []byte `json:"decrypterKeyPEM,omitempty"`
+	DecrypterKeyURI      string `json:"decrypterKey,omitempty"`
+	DecrypterKeyPassword []byte `json:"decrypterKeyPassword,omitempty"`
```

Comment:
We might have users using a release candidate where this is a `string`. I would prefer to use `[]byte` for this. But custom `Password` type that can have either base64 or plain passwords can be considered.

Reply:
People may have been using it, but those changes were only in a (draft) PR until now, so I don't think we need to spend a lot of time on keeping this compatible. I know of at least one user who tested this with the `decrypterKey` with `cloudkms`, and that doesn't need the password.